src/cljs/athens/views/buttons.cljs

```diff
@@ -23,41 +38,49 @@
-                               :cursor "default"}]]
-   ::stylefy/manual [[:svg {:margin-block-start "-0.0835em"
-                            :margin-block-end "-0.0835em"}
-                      [(selectors/& (selectors/not (selectors/last-child))) {:margin-inline-end "0.251em"}]
-                      [(selectors/& (selectors/not (selectors/first-child))) {:margin-inline-start "0.251em"}]
-                      [(selectors/& ((selectors/first-child (selectors/last-child)))) {:margin-inline-start "-4px"
-                                                                                       :margin-inline-end "-4px"}]]
+   :transition       "all 0.075s ease"
+   ::stylefy/manual [[:&:hover {:background (darken (color :panel-color :opacity-low) 10)}]
+                     [:&:active
+                      :&:hover:active
+                      :&.active {:color (color :body-text-color)
+                                 :background-color  (darken (color :panel-color :opacity-med) 10)}]
+                     [:&:disabled {:color (color :body-text-color 0.3)
+                                   :background-color (color :panel-color :opacity-higher)
+                                   :cursor "default"}]
                      [:span {:flex "1 0 auto"
                              :text-align "left"}]
-                     [:&.active {:background-color (darken (color :panel-color) 10)}]]})
+                     [:.MuiSvgIcon-root button-icons-style
+                      [(selectors/& (selectors/not (selectors/last-child))) button-icons-not-last-child-style]
+                      [(selectors/& (selectors/not (selectors/first-child))) button-icons-not-first-child-style]
+                      [(selectors/& ((selectors/first-child (selectors/last-child)))) button-icons-only-child-style]]]})
 
 
 (def buttons-primary-style
-  (merge buttons-style {:color "rgba(0, 117, 225)"
-                        :background-color "rgba(0, 117, 225, 0.1)"
-                        ::stylefy/mode [[:hover {:background-color "rgba(0, 117, 225, 0.25)"}]
-                                        [:active {:color "white"
-                                                  :background-color "rgba(0, 117, 225, 1)"}]
-                                        [:disabled {:color "rgba(0, 0, 0, 0.3)"
-                                                    :background-color "#EFEDEB"
-                                                    :cursor "default"}]]}))
+  (merge buttons-style {:color (color :link-color)
+                        :background-color (color :link-color :opacity-lower)
+                        ::stylefy/manual [[:&:hover {:background (color :link-color :opacity-low)}]
+                                          [:&:active
+                                           :&:hover:active
+                                           :&.active {:color "white"
+                                                      :background-color (color :link-color)}]
```

Comment:
We need to have `&:active` and `&.active` pseudo and class selectors?

Reply:
One is for when the browser sees the user is clicking, one is for toggling the 'pressed' state of the button, such as when it’s indicating the active nav location. Renamed to `.is-active` for consistency.